Saule/JsonApiSerializer.cs

```diff
@@ -72,7 +72,9 @@ public PreprocessResult PreprocessContent(object @object, ApiResource resource,
                     type: resource,
                     baseUrl: requestUri,
                     urlBuilder: UrlPathBuilder,
-                    paginationContext: QueryContext?.Pagination);
+                    paginationContext: QueryContext?.Pagination,
+                    includingContext: QueryContext?.Including,
+                    includedDefault: QueryContext != null ? QueryContext.IncludedDefault : true);

```

Comment:
If I understand you correctly (I don't quite understand the "may always be null" part..) that would require the parameter/member includedDefault in ResourceSerializer to be a nullable boolean, isn't it better to coalesce the QueryContext.IncludedDefault before that as done here?